service/controller/app/v1/resource/chartoperator/resource.go

```diff
@@ -89,7 +89,7 @@ func (r Resource) Name() string {
 	return Name
 }
 
-func (r Resource) installChartOperator(ctx context.Context, cr v1alpha1.App) error {
+func (r Resource) installChartOperator(ctx context.Context, cr v1alpha1.App, update bool) error {
```

Comment:
`update` parameter added since we want to distinguish between initial install and retrial.